kyaml/runfn/runfn.go

```diff
@@ -101,6 +101,10 @@ type RunFns struct {
 	// If it is true, the empty result will be provided as input to the next
 	// function in the list.
 	ContinueOnEmptyResult bool
+
+	// WorkDir specifies which working directory an exec function should run in.
+	// If this is empty, fall back to the current working directory.
+	WorkDir string
```

Comment:
Why `WorkDir` here and `WorkingDir` everywhere else?

Reply:
an oversight, I changed it!